web/src/pages/HomePage/HomePage.tsx

```diff
@@ -65,4 +75,74 @@ const HomePage = () => {
   )
 }
 
+export const IsMilestone = (page: string, milestone: string) => {
```

Comment:
This function should not be exported from the HomePage, it should be exported from its own file.